tests/PathIdentificationTest.php

```diff
@@ -123,3 +127,17 @@
         ->withoutExceptionHandling()
         ->get('/acme/foo/abc/xyz');
 });
+
+test('tenant set as a default parameter for the URLs', function () {
```

Comment:
Fix the grammar here please

Reply:
Grammar looks good, but not sure about the test name overall. Renaming it to `tenant parameter is set for all routes as the default parameter once the tenancy initialized`

Reply:
> Grammar looks good

It's not, it says `tenant set` rather than `tenant is set`. Also I'd change `the URLs` to `URLs` since we aren't referring to any specific URLs.

Anyway, I can rewrite this myself, just wanted to point it out.